code/modules/hydroponics/plant_genes.dm

```diff
@@ -408,7 +439,19 @@
 	name = "Hypodermic Prickles"
 
 /datum/plant_gene/trait/stinging/on_slip(obj/item/reagent_containers/food/snacks/grown/G, atom/target)
-	on_throw_impact(G, target)
+	if(isliving(target) && G.reagents && G.reagents.total_volume)
```

Comment:
Could you explain why you aren't just using throw_impact to do the logging?

Reply:
Because throw is a throw and it gets logged as combat.